src/_scss/core/_variables.scss

```diff
@@ -201,17 +199,17 @@ $colors-grants-nff: #47AAA7;
 $colors-loans-subsidy: #8c6e86;
 $colors-loans-face-value: #ded5db;
 
-$colors-obligated: #4773aa;
+$colors-obligated: $color-cool-blue-light;
 $colors-current: #8ba6c9;
 $colors-potential: #dce4ee;
 $colors-outlayed: #0b2e5a;
 
 $color-infrastructure-obligations: #afcdd5;
 $color-infrastructure-outlays:    #2d6878;
-
-$headline-color: #1b1b1b;
-$body-color: #5c5c5c;
-$card-outline-color: #dfe1e2; // gray-cool-10 from USWDS
+///
+$headline-color: $gray-90;
+$body-color: $gray-60;
+$card-outline-color: $gray-cool-10; // gray-cool-10 from USWDS
```

Comment:
this comment is now outdated, since we're using gray-cool-10 in the definition now